scripts/get

```diff
@@ -207,6 +207,10 @@ while [[ $# -gt 0 ]]; do
        shift
        if [[ $# -ne 0 ]]; then
            export DESIRED_VERSION="${1}"
+           if [[ "$1" != "v"* ]]; then
```

Comment:
This will fail if the version is `latest`

Reply:
Wanted to make note that the current install script on `main` will fail if you pass `--version latest` to it as well, so this doesn't appear to be a change in behavior. The default behavior of the script to fetch the `latest` version by default when no `--version` flag is passed in is still functioning as expected. These changes appear to fix the issue that was raised.

Reply:
Adding a check like: `&& [[ "$1" != "latest" ]]` to the if statement hushes the output that notifies the user that it's fixing the version input when it fails. It may be less confusing to hush the output when it does fail